src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Checkbox from 'src/shared/checkbox/Checkbox';
+
+import styles from './CheckBoxGrid.scss';
+
+type Props = {
+  gridData: any;
```

Comment:
Can this be more type-specific (at least to the level that it's an object or an object containing nested objects)?

Reply:
That's done.